en/mkdocs.yml

```diff
@@ -672,7 +672,6 @@ nav:
             - 'Changing to Remote H2': setup/changing-to-remote-h2.md
             - 'Changing default datasource of BPS': setup/changing-datasource-bpsds.md
             - 'Changing the Default Datasource for Consent Management': setup/changing-datasource-consent-management.md
```

Comment:
Have we updated the `About this Release` page?
Shall we add a section called `Removed features`:
https://is.docs.wso2.com/en/5.12.0/get-started/about-this-release/#deprecated-features

Reply:
@ImalshaG